docker-compose-gpu-runner-cuda-12-0.yml

```diff
@@ -1,14 +1,27 @@
 version: '3.2'
 services:
-  geth:
-    image: "matterlabs/geth:latest"
-    environment:
-      - PLUGIN_CONFIG
+  reth:
+    restart: always
+    image: "ghcr.io/paradigmxyz/reth:v0.2.0-beta.1"
+    volumes:
+      - type: bind
+        source: ./volumes/reth/data
+        target: /rethdata
+      - type: bind
+        source: ./volumes/reth/logs
+        target: /rethlogs
```

Comment:
Do we need to bind it? IIRC for non-mounted directories, an ephemeral storage would be created that is dropped once the container exits. This seems to be fine for reth logs -- I doubt we'll have to check them often.